cluster/gce/gci/configure-helper.sh

```diff
@@ -1423,6 +1423,9 @@ function start-kube-addons {
   if [[ "${ENABLE_DEFAULT_STORAGE_CLASS:-}" == "true" ]]; then
     setup-addon-manifests "addons" "storage-class/gce"
   fi
+  if [[ -n "${NON_MASQUERADE_CIDR:-}" == "0.0.0.0/0" ]]; then
```

Comment:
I don't know bash in this much detail, how does this if statement evaluate?
`(-n "${NON_MASQUERADE_CIDR:-}") == "0.0.0.0/0"`? because `True != "0.0.0.0/0"`...
`(-n "${NON_MASQUERADE_CIDR:-}") && ("${NON_MASQUERADE_CIDR:-}"  == "0.0.0.0/0")`?